docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -0,0 +1,304 @@
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/module_settings.png
+    :align: center
+    :scale: 50
+    :alt: All Modules Set Correctly
+
+    All Modules Set Correctly
+
+
+Actuator Setup
+==============
+
+Now the actuators must be set up in the actuator tab of QGroundControl. Navigate to the actuator tab of QGRoundControl and ensure that the Vertiq IO tab shows up.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/actuator_vertiq_tab.png
+    :align: center
+    :scale: 50
+    :alt: Vertiq IO Tab
+
+    Vertiq IO Tab Appears
+
+The geometry section determines where each motor is on the aircraft and what direction it is spinning. The Actuator Outputs section determines which motor defined in the geometry section is connected to which actuator. In this case we want to associate Motors from the geometry section with our Modules. The ESCs in the Actuator Outputs section correspond to the CVIs set in the previous section, but are offset by 1. If you would like to set the Motor associated with CVI 0, you need to set that motor on ESC 1, for CVI 1, you must set ESC 2, etc. In this example we have set the CVIs such that we can associate Motor 1 with ESC 1, Motor 2 with ESC 2, Motor 3 with ESC 3, and Motor 4 with ESC 4.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/actuator_associations.png
+    :align: center
+    :scale: 50
+    :alt: Motor to Actuator to CVI Associations
+
+    Motor to Actuator to CVI Associations
+
+If you accidentally put the wrong CVI on a motor you can just switch the motor functions in the actuator outputs section. If for example you accidentally set the front right motor as CVI 2 and the front left motor as CVI 0, then you can just set ESC 1 as Motor 3, and ESC 3 as Motor 1. You could also change the CVIs associated with each module, but we find that this is the most straightforward way to swap connections.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/actuator_associations_swap.png
+    :align: center
+    :scale: 50
+    :alt: Motor to Actuator to CVI Associations with Swap
+
+    Motor to Actuator to CVI Associations with Swap
+
+Spin Testing
+============
+
+.. warning::
+    Please remove all propellers from any module you plan on testing. Failure to do so can result in harm to you or others around you. Further, please ensure that your module is secured to a stationary platform or surface before attempting to spin it. 
+
+Now that the modules and flight controller have been configured, the modules can be tested. This is done in the same Actuators Tab in QGroundControl with the Actuator Testing section.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/sliders.png
+    :align: center
+    :scale: 50
+    :alt: Actuator Testing
+
+    Actuator Testing Section
+
```

Comment:
I would love a video of this working...but that might mean I have to do it because you do not have the light box...

Reply:
Your call? Can we embed videos easily?

Reply:
Yea we can, but I don't have a great way to make a nice vehicle, so I'm going to punt on this for now.